infra/helper.py

```diff
@@ -713,10 +716,23 @@ def build_fuzzers_impl(  # pylint: disable=too-many-arguments,too-many-locals,to
 
 def build_fuzzers(args):
   """Builds fuzzers."""
+  if args.engine == 'centipede':
+    # A hack to place target binary (with san) into a subdir of project.
+    project_name = args.project.name
+    args.project.name += f'/{args.project.name}_{args.sanitizer}'
```

Comment:
Rather than overwriting `project.name`, which is quite hacky, can we pass this as a separate argument to `build_fuzzers_impl`?

Reply:
Yep, that is better indeed.